src/main/java/sc/fiji/bdvpg/bdv/navigate/ViewerTransformSyncStarter.java

```diff
@@ -0,0 +1,90 @@
+package sc.fiji.bdvpg.bdv.navigate;
+
+import bdv.util.BdvHandle;
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.ui.TransformListener;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Action which synchronizes the display location of n BdvHandle
+ *
+ * Works in combination with the action ViewerTransformSyncStopper
+ *
+ * See also ViewTransformSynchronizationDemo
+ *
+ * author Nicolas Chiaruttini, BIOP EPFL, nicolas.chiaruttini@epfl.ch
+ *
+ */
+
+public class ViewerTransformSyncStarter implements Runnable {
+
+    BdvHandle[] bdvhs;
+
+    BdvHandle originatingBdvHandle = null;
+
+    Map<BdvHandle, TransformListener<AffineTransform3D>> bdvToTransformListener = new HashMap<>();
+
+    public ViewerTransformSyncStarter(BdvHandle[] bdvhs) {
+       this.bdvhs = bdvhs;
+    }
+
+    public void setOriginatingBdvHandle(BdvHandle bdvh) {
+        originatingBdvHandle = bdvh;
+    }
+
+    @Override
+    public void run() {
```

Comment:
The code in this function is quite long. Could you try to refactor it in a sequence of smaller functions?
It would make it easier to read.

Reply:
We could also merge it and then I can try to do this, thereby checking whether I understand what is going on.

Reply:
refactored and more documented in https://github.com/bigdataviewer/bigdataviewer-playground/pull/68/commits/72739e1cf201b698fe4dbfb38b982d5515b2e1a4